Source/ForecastDeskbarView.cpp

```diff
@@ -96,14 +98,19 @@ void
 ForecastDeskbarView::MessageReceived(BMessage* message)
 {
 	if (message->what == kUpdateForecastMessage) {
+
+		// Todo: Reload after X minutes
+		//if (fForecastView->IsConnected())
+			//fForecastView->Reload();
+
 		BString weatherDetailsText;
-		weatherDetailsText << "Temperature: "
+		weatherDetailsText << B_TRANSLATE("Temperature: ")
```

Comment:
Here, and for rhe strings below, I suggest to remove the space at the end of the string and append it separately. A space at the end can easily be overlooked by the translators, so:
```suggestion
		weatherDetailsText << B_TRANSLATE("Temperature:") << " "
```
Or go with one long translatable string that contains variables ("%temperature%" etc.)  that get replaced using BString's SetToFormat() etc.

Reply:
Added a single, translatable string that inserts the variables with SetToFormat() as suggested.